### ISSUE TITLE
Duplicate entries in ISM line list

### ISSUE THREAD

**Issue by nhmc** [linetools/linetools#16]
There are duplicate entries in the ISM line list:

```python
ll = linetools.lists.Linelist('ISM')
for n in ll._data['name']:
    if (ll._data['name'] == n).sum() > 1:
        print(n)
```
shows CII* 1280a and FeII* 2396 duplicated. Will look at this in more detail later...

**Comment by ntejos** [comment]
I noticed this bug too, which is more evident when one load linelists from
multiple lists. Will try to fix it today.

**Comment by profxj** [comment]
Took a quick look and see nothing obvious (although I agree
it is a ‘bug’).  Good luck with this one!

X

On Jul 2, 2015, at 7:00 AM, Nicolas Tejos <notifications@github.com> wrote:

> I noticed this bug too, which is more evident when one load linelists from
> multiple lists. Will try to fix it today.
> On Jul 2, 2015 5:21 AM, "Neil Crighton" <notifications@github.com> wrote:
> 
> > There are duplicate entries in the ISM line list:
> >
> > ll = linetools.lists.Linelist('ISM')for n in ll._data['name']:
> > if (ll._data['name'] == n).sum() > 1:
> > print(n)
> >
> > shows CII* 1280a and FeII* 2396 duplicated. Will look at this in more
> > detail later...
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/linetools/linetools/issues/16>.
> >
> —
> Reply to this email directly or view it on GitHub.
> 
> !DSPAM:826,559543ee12972215511037!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/profxj/xastropy

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)

**Comment by nhmc** [comment]
Closed by #17
